tests/unix_serial_tests.cc

```diff
@@ -20,8 +20,6 @@ void loop()
 #include <string>
 #include "gtest/gtest.h"
 
-#include <boost/bind.hpp>
-
```

Comment:
I think this can be removed from the package.xml too

Reply:
hm...
    
    $ git grep bind.hpp
    tests/proof_of_concepts/tokenizer.cc:#include <boost/bind.hpp>

Reply:
however, 2070f8ab683af8f4 gets rid of this dep

Reply:
The `tests/proof_of_concepts` directory seems to be dead code that's not being built